src/spiceypy/spiceypy.py

```diff
@@ -5609,6 +5632,51 @@ def getfov(
     )
 
 
+@spice_error_check
+def getfvn(
+    inst: str,
+    room: int,
+    shapelen: int = _default_len_out,
```

Comment:
I don't think `shapelen` or `framelen` are needed, assuming that `_default_len_out` is enough to fit any shape name or frame name. By providing these input arguments, you're giving the users the option to set an invalid length and get a truncated output.

Reply:
so far my policy has been, for the most part, to allow overrides in this manner. these are default values so this isn't a significant concern, and GIGO always applies, although it could be a good opportunity to use keyword-only args (pep3102) to provide a bit more a guard without preventing user interaction with these parameters